frontend/src/pages/Mesh/target/TargetPanelControlPlane.tsx

```diff
@@ -260,6 +264,20 @@ export class TargetPanelControlPlane extends React.Component<
       });
   };
 
+  private fetchCertificates = async (): Promise<void> => {
+    const data = this.state.controlPlaneNode!.getData() as NodeData;
+    console.log(data);
+    return API.getIstioCertsInfo()
+      .then(response => {
+        this.setState({
+          certificates: response.data
+        });
+      })
+      .catch(error => {
+        AlertUtils.addError('Error fetching namespace certificates.', error, 'default', MessageType.ERROR);
+      });
+  };
```

Comment:
Since the certificates are already coming from `data.infraData.config.certificates`, do we need this API call / endpoint at all?